docs/howto/danceable-lyrics-input-tool/README.md

```diff
@@ -0,0 +1,561 @@
+# Danceable Lyrics
+In this guide, we use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create a tool that connects directly to a set of data files and determines the most "danceable" songs that contain a set of lyrics. Perhaps we started this journey with the following problem statement:
+> I am a Spotify Product Analyst who wants to perform a POC on the feasibility of determining the most "danceable" songs that contain a set of lyrics. If possible this POC can help drive the development of a new Recommendation Feature on the Spotify App.
+
+We use the [Polars](https://www.pola.rs/) for data processing. Polars has many benefits over the more common Pandas, not limited to:
```

Comment:
TW review:

We use [Polars](https://www.pola.rs/) for data processing. Polars has many benefits over the more common Pandas. Some key benefits include: